Makefile

```diff
@@ -1,60 +1,67 @@
-VENV = venv
-PYTHON = $(VENV)/bin/python3
```

Comment:
This `PYTHON` only works if you already have a `venv` folder, so you are in a bit of a chicken and egg problem.

I changed it instead to use the system `python` to bootstrap the `venv` creation.